Sources/Redis/Client/RedisClient+KeyedCache.swift

```diff
@@ -1,20 +1,31 @@
+//
+//  RedisClient+KeyedCache.swift
+//  Async
+//
+//  Created by Shaun Hubbard on 3/9/18.
```

Comment:
no file headers. (your name will be in the commits and as a contributor on gh :) )

Reply:
D'oh... Xcode... I really should take some time and write a VaporPlugin for Emacs....

Updated Priorities Lists 
1. VGSM - API -  Deployment to Vapor Cloud???
2. Vapor/Services - Draft Life Cycle RFC
3. Vapor/AMQP
4. VGSM Client(s) iOS/Web/Droid
5. Emacs Plugin For Vapor